altair/examples/scatter_linked_table.py

```diff
@@ -0,0 +1,57 @@
+"""
+Brushing Scatter Plot to show data on a table
+---------------------------------
+A scatter plot of the cars dataset, with data tables for horsepower, MPG, and origin. 
+The tables update toreflect the selection on the scatter plot.
+"""
+# category: scatter plots
+
+import altair as alt
+from vega_datasets import data
+
+source = data.cars()
+
+# Brush for selection
+brush = alt.selection(type='interval')
+
+# Scatter Plot
+points = alt.Chart().mark_point().encode(
+    x='Horsepower:Q',
+    y='Miles_per_Gallon:Q',
+    color=alt.condition(brush, 'Cylinders:O', alt.value('grey'))
+).add_selection(brush)
+
+# Base chart for data tables
+ranked_text = alt.Chart().mark_text().encode(
+    y=alt.Y('row_number:O',axis=None)
+).transform_window(
+    row_number='row_number()'
+).transform_filter(
+    brush
+).transform_window(
+    rank='rank(row_number)'
+).transform_filter(
+    alt.datum.rank<20
+)

```

Comment:
I think this can be accomplished with one less transform: 
```python
).transform_filter(
    brush
).transform_window(
    row_number='row_number()'
).transform_filter(
    alt.datum.row_number<20
)
```

Reply:
I think you're right. I was following the [Vega-Lite example](https://vega.github.io/vega-lite/examples/brush_table.html) somewhat literally and was nervous that I had missed some edge case where you couldn't just use the row_number, but I think it's fine so long row_number is set after the brush filter is applied.